openapi/openapi.yaml

```diff
@@ -246,4 +245,64 @@ paths:
         '200':
           description: The current User
         'default':
-          description: Error
+          $ref: '#/components/responses/Error'
+
+components:
+  schemas:
+    Error:
```

Comment:
ErrorList name would be more appropriate, as it's a list of an individual error items.

Reply:
Also if Error schema is used only in Error response schema, maybe it's reasonable to merge them?

Reply:
The top level Error object calls the field name 'errors' so chose 'Errors' to reflect that.

Reply:
re combining Error schema / Error response

There can be other error responses that need to ref an Error schema.
For example, a 'UnauthorizedResponse'  for 401 that returns the same Error schema but adds
a 'WWW-Authenticate' header.

Reply:
Ah, 'Error' vs 'Errors'!  good catch

Reply:
updated

Reply:
Also,  went with 'Errors' since it matches the field name 'errors' in the top level response (https://jsonapi.org/format/#document-top-level)